### PR TITLE
Integrated details and inventory cards in storage overview page

### PR DESCRIPTION
**Pull request by gnehapk**

### CODE REVIEW

**Comment by gnehapk** [comment]
@mareklibra @rawagner Please review.

File: frontend/public/storage/components/storage-overview/storage-overview.jsx [hunk 1]
```diff
@@ -1,16 +1,80 @@
 import React from 'react';
+import {
+  StorageOverview as KubevirtStorageOverview,
+  ClusterOverviewContext,
```
Comment (rawagner) [inline frontend/public/storage/components/storage-overview/storage-overview.jsx]: you should create your own context ie `StorageOverviewContext`

**Comment by cloudbehl** [comment]
Closing PR: Opened a new PR after the change. https://github.com/kubevirt/web-ui/pull/260

**Comment by rawagner** [comment]
Closing this one as it is superseded by https://github.com/kubevirt/web-ui/pull/260

**Pull request closed without merging**